ansible/roles/baselayout/vars/main.yml

```diff
@@ -106,6 +106,6 @@ packages: {
   ],
 
   ubuntu1404: [
-    'ntp,g++-4.8,gcc-4.8,g++-4.9,gcc-4.9,binutils-2.26',
+    'ntp,gcc-6,g++-6,g++-4.8,gcc-4.8,g++-4.9,gcc-4.9,binutils-2.26',
```

Comment:
switching order might make the last installed the default, but if gcc-4.9 is already installed, YMMV